src/main/java/org/opentripplanner/service/realtimevehicles/model/RealtimeVehicleBuilder.java

```diff
@@ -20,66 +21,73 @@ public class RealtimeVehiclePositionBuilder {
   private StopStatus stopStatus = StopStatus.IN_TRANSIT_TO;
   private StopLocation stop;
   private Trip trip;
+  private OccupancyStatus occupancyStatus;
 
-  public RealtimeVehiclePositionBuilder setVehicleId(FeedScopedId vehicleId) {
```

Comment:
I wrote this builder at a time when we didn't have well-establised conventions for builders. Could you please rename the methods to `withX`?